docs/dabp/guides/technical/create-resource-type-via-api.md

```diff
@@ -0,0 +1,163 @@
+# Creating resource type via API
+Resource types can be created only via API as they should not change after initial setup of the platform.
+
+## How to create a resource type
+In this guide, we will assume Onegini Identity Cloud is available under the URL: `https://dabp.onegini.com`.
+The API is protected with OIDC, so you need to provide an access token to make a successful request.
+Please note that the access token must be linked to a person who has the `Add/edit/delete resources and resource types` permission 
+on the root group. The root group is the top-level group.
+
+Resource type can be linked to a policy. If a resource type is linked to policy, then all resources of that type can be assigned only to groups/members if they also have this policy assigned.
+You can think that a policy on a resource type acts as a filter to whom a resources of this type will be available.
+If resource type is created without a policy id, then all resources of this type will be available to all groups and members 
+
+To create a resource type a POST request must be sent to `https://dabp.onegini.com/delegation/api/v2/resource-types` with a proper request body
+```
+{
+    "name": "Life portfolio type",
+    "policyId": <policy id required to access resources of this type [optional]>
```

Comment:
required and optional in the same line looks strange, maybe change to "needed"

Reply:
fixed